test/integration/component/test_project_usage.py

```diff
@@ -221,10 +219,35 @@ def test_01_vm_usage(self):
         #    VM.Destroy and volume .delete Event for the created account
         # 4. Delete the account
 
+        projectlist=Project.list(self.apiclient,account=self.account.name,
+                                 domainid=self.account.domainid,id=self.project.id)
+
+        self.assertEqual(projectlist[0].vmrunning,0,"vmrunning value is not returned")
```

Comment:
Before checking the no.of running and stopped vms, check whether the projectlist has vmrunning and vmstopped attributes.